src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball{
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors colors = Colors.values()[index];
+        setColor(colors.toString());
+        return colors.toString();
```

Comment:
You are calling `toString()` on the enum, which is not recommended. Use `name()` to get the String representation of the enum constants.